terraform/oke/schema.yaml

```diff
@@ -111,47 +100,61 @@ variables:
   #   required: true
 
   ####
-  ##  OCI Observability & Management Services Configuration
+  ##  OCI Observability and Management Services Configuration
   ####
-  
+
+  #  Stack Deployment Options
+  stack_deployment_option:
+    title: Deployment options
+    description: 'The "Full" option creates both OCI and K8s resources.'
+    type: enum
+    enum: # Dev Note - # Any change in following options must be refactored across schema.yaml
+    - "Full"
+    - "Only OCI Resources"
+    default: "Full"
+    required: true
+    visible:
+       not: 
+       - livelab_switch    
+
   # Option to enable/disable metric server installation during helm deployment
   opt_deploy_metric_server:
     type: boolean
-    title: Enable Metric Server Installation
-    description: Uncheck this if Metric Server is already installed in your cluster.
+    title: Enable Metric Server installation
```

Comment:
Deploy Kubernetes Metric Server

Description: Kubernetes Metrics Server is required for scraping metrics. Select this checkbox if this OKE Cluster doesn't have Kubernetes Metrics Server deployment.